src/aliases/manager/manager.go

```diff
@@ -0,0 +1,50 @@
+package aliasmanager
```

Comment:
I cannot understand the propose of having a AliasManager. Managers are long living instances of our application which are intended for holding a pice of the application state. For instance in the case of nodes and stores, the manager is notified about manifest content in order to instanciate Stores and Nodes services which are the ones connected to the DB and KeyVaults but the manager never interacts with any of those directly as it is your case here.

In your case, since you don't have any state to be held I would rather propose to initialize an AliasConnector (just to follow same convention than Stores) instead of the manager and that service is the one being passed to your handlers to interact with the db

Reply:
I need to have a Runnable to register it, even though it doesn't deal with any state.
Saw that suggestion with @nmvalera and we will keep as is.

Reply:
Finally removed it after a discussion with @ggarri.
The service doesn't need to be registered this way as the routes are registered via the mux.Router.

But now, if I need to check Liveness/Readiness checks, I guess I need to re-create a Runnable?
Let's do that in another small PR, shall we?